app-config.json

```diff
@@ -933,6 +933,183 @@
+                    "type": ["Item"],
+                    "id": "https://id.gs1.org/01/09520123456788/21/12345",
+                    "name": "EV battery 300Ah."
+                  },
+                  "childEPCList": [
+                    {
+                      "type": ["Item"],
+                      "id": "https://id.gs1.org/01/09520123456788/21/12345",
+                      "name": "EV battery 300Ah."
+                    },
+                    {
+                      "type": ["Item"],
+                      "id": "https://id.gs1.org/01/09520123456788/21/12345",
+                      "name": "EV battery 300Ah."
+                    }
+                  ],
+                  "childQuantityList": [
+                    {
+                      "productId": "https://id.gs1.org/01/09520123456788/21/12345",
+                      "productName": "EV battery 300Ah.",
+                      "quantity": 20,
+                      "uom": "KGM"
+                    },
+                    {
+                      "productId": "https://id.gs1.org/01/09520123456788/21/12345",
+                      "productName": "EV battery 300Ah.",
+                      "quantity": 20,
+                      "uom": "KGM"
+                    }
+                  ]
+                },
+                "constructData": {
+                  "mappingFields": [],
+                  "dummyFields": [],
+                  "generationFields": [
+                    {
+                      "path": "/eventID",
+                      "handler": "generateIdWithSerialNumber"
+                    }
+                  ]
+                },
+                "className": "json-form",
+                "style": {
+                  "margin": "40px auto",
+                  "paddingTop": "40px",
+                  "width": "80%"
+                }
+              }
+            },
+            {
+              "name": "CustomButton",
+              "type": "Submit",
+              "props": {}
+            }
+          ],
+          "services": [
+            {
+              "name": "processAssociationEvent",
+              "parameters": [
+                {
+                  "vckit": {
+                    "vckitAPIUrl": "http://localhost:3332/v2",
+                    "issuer": "did:web:uncefact.github.io:project-vckit:test-and-development"
+                  },
+                  "epcisAssociationEvent": {
+                    "context": [
+                      "https://jargon.sh/user/unece/traceabilityEvents/v/0.5.0/artefacts/jsonldContexts/traceabilityEvents.jsonld?class=traceabilityEvents"
+                    ],
+                    "renderTemplate": [
+                      {
+                        "template": "<div><h2>Association Event</h2></div>",
+                        "@type": "WebRenderingTemplate2022"
+                      }
+                    ],
+                    "type": ["VerifiableCredential", "DigitalTraceabilityEvent"],
```

Comment:
I think using this array data type might result in the issued VC (output of issueVC function) having a type like: ["VerifiableCredential", "VerifiableCredential", "DigitalTraceabilityEvent"]. This would cause a duplication of `"VerifiableCredential"`.

Reply:
Yep, fixed.